key_management/src/keystore.rs

```diff
@@ -242,19 +249,14 @@ impl KeyStore for PersistentKeyStore {
             return Err(Error::KeyExists);
         }
 
-        let passphrase = String::from("default");
-        let generated_key = PersistentKeyStore::generate_key(&passphrase)?;
-
         self.key_info.insert(key, key_info);
-        let mut file = OpenOptions::new()
+        let file = OpenOptions::new()
             .write(true)
             .create(true)
             .open(&self.location)
             .map_err(|err| Error::Other(err.to_string()))?;
-        let json =
-            serde_json::to_string(&self.key_info).map_err(|err| Error::Other(err.to_string()))?;
-        let encrypted_json = PersistentKeyStore::encrypt(&generated_key, json.as_bytes())?;
-        file.write_all(&encrypted_json)?;
+        serde_json::to_writer(&file, &self.key_info)
```

Comment:
Do you think we could use serde_cbor here instead?

Reply:
Makes sense to json this. Its going into a file and a user might wanna inspect it or something. Generally id say we should use cbor for when we wanna store in DB or send over the wire.

Reply:
We're actually using a separate file for the encrypted keystore (without the `.json` extension), and the entire file is encrypted bytes, so I think it makes more sense to serialize the data to bytes, also.

Reply:
So we don't want to use CBOR? Just keep everything as JSON and encrypt the data as a whole?

Reply:
I think CBOR is better for the encrypted keystore, but JSON is better for the plaintext JSON file